src/ajax.js

```diff
@@ -19,6 +20,7 @@ var
 	// #7653, #8125, #8152: local protocol detection
 	rlocalProtocol = /^(?:about|app|app-storage|.+-extension|file|res|widget):$/,
 	rnoContent = /^(?:GET|HEAD)$/,
+	rformContent = /x-www-form-urlencoded/,
```

Comment:
If we're testing by regex, this should be `/^application\/x-www-form-urlencoded[\x20\t]*(?:;.*|)$/` for maximum [spec](https://tools.ietf.org/html/rfc7231#section-3.1.1.1) adherence. Equivalently, we could test by `(s.contentType || "").split( /[\x20\t;]+/ )[ 0 ] === "application/x-www-form-urlencoded"`. But in either case, both the below test and [jsonp](https://github.com/dmethvin/jquery/blob/2658-form-encoding/src/ajax/jsonp.js#L28-L29) should use the same logic.